### PR TITLE
chore: delete superseded TRG 3-1

### PR DESCRIPTION
**Pull request by FaGru3n**
## Description

- delete superseded TRG 3-1 "Kubernetes Version"
- add hint to TRG Helm Test with kubernetes version 91ce37f6bca1f35bdc4e3e0772e2bb548c4fe9bf
- add redirect for deleted 3-1 link to general Release Guidelines

fixes https://github.com/eclipse-tractusx/sig-infra/issues/405

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by SebastianBezold — DISMISSED** [review]
👍 . I think you can remove the "and renumber TRG 3-2" from the tile

File: docs/release/trg-3/trg-3-1.md
Comment (carslen) [inline docs/release/trg-3/trg-3-1.md:1]: I'm struggeling with deleting the TRG without mention it anywhere. Instead I'd leave the TRG with following content (and rephrased Title):

```text
---
title: TRG 3.01 Supported Versions - Superseded
---
| Status     | Created      | Post-History         |
|------------|--------------|----------------------|
| Superseded | 08-Mar-2023  | Marked as superseded |
| Moved      | 02-Jan-2023  | content moved        |
| Draft      | 13-Sept-2022 | n/a                  |

## Description

This TRG has been superseeded by [TRG 5.09 - Helm Test](../trg-5/trg-5-09).
```

Comment (FaGru3n) [reply docs/release/trg-3/trg-3-1.md:1]: Thanks for feedback, little question from myside is your idea more related to inform everyone or keeping the document as informational note?

Guess for feedback or information i can also create a little disscussion and inform everyone on the mailing list.

Do you agree or what do you think?

Comment (Siegfriedk) [reply docs/release/trg-3/trg-3-1.md:1]: I think, to a certain degree, it should be kept (old links etc.)

Comment (carslen) [reply docs/release/trg-3/trg-3-1.md:1]: Having a hole in the list of available TRGs could lead to questions. I know that the adjustments to the docusurus config will redirect people accessing this TRG from a bookmark, but to me that that would confuse me too. I'd keep the TRG with related informations. Or is there another changelog which mentions the deletion of this TRG? At least this kind of information is not part of this PR.

Comment (FaGru3n) [reply docs/release/trg-3/trg-3-1.md:1]: Thank you very much for the input, i would like to follow the deletion (like the TRG 2.02) but with your hints updated also the "Changelog & Drafts" Section then we could keep the old links in the GitHub History and sharing a bit the information why we deleted it.
[727a5e2](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/618/commits/727a5e20892e64eb2526cee4efbfe842851f2d58)

**Review by hzierer — APPROVED** [review]

**Review by SebastianBezold — APPROVED** [review]
I think cleaning up by actually removing is the better option, than keeping it to keep numbering straight.
Linking to a revision in the history seems a good option to me

**Review by tomaszbarwicki — APPROVED** [review]
LGTM

**Pull request merged**